data/json/monsters/turrets.json

```diff
@@ -101,17 +131,25 @@
     "armor_cut": 16,
     "vision_night": 3,
     "revert_to_itype": "bot_rifleturret",
-    "starting_ammo": { "556": 100 },
+    "starting_ammo": { "556": 1600 },
     "special_attacks": [
       {
         "type": "gun",
         "cooldown": 1,
-        "gun_type": "m4a1",
+        "move_cost": 150,
+        "gun_type": "m249",
         "ammo_type": "556",
-        "fake_skills": [ [ "gun", 6 ], [ "rifle", 8 ] ],
-        "fake_dex": 10,
-        "range": 18,
-        "ranges": [ [ 0, 10, "AUTO" ], [ 11, 30, "DEFAULT" ] ]
+        "fake_skills": [ [ "gun", 8 ], [ "rifle", 8 ] ],
+        "fake_dex": 12,
+        "ranges": [ [ 0, 30, "DEFAULT" ] ],
+        "require_targeting_npc": true,
+        "require_targeting_monster": true,
```

Comment:
Is there a reason it doesn't do this for players too?

Reply:
`gun_actor::require_targeting_player` is initialized to true so it only needs setting in the attack definition if you want to turn it off.

Reply:
Ah, I didn't see anything saying that in the documentation, I think I'll update that.